sygnal/gcmpushkin.py

```diff
@@ -452,11 +452,14 @@ def __init__(self):
 
         Args:
             db (adbapi.ConnectionPool): database to prepare
+            engine (str):
+                Database engine in use. This is either "sqlite" or "postgresql".
```

Comment:
it's not in use yet?

```suggestion
                Database engine to use. Shoud be either "sqlite" or "postgresql".
```